pallets/parachain-staking/src/tests.rs

```diff
@@ -5395,6 +5395,9 @@ fn payouts_follow_delegation_changes() {
 		])
 		.build()
 		.execute_with(|| {
+			// ~ set block author as 1 for all blocks this round
+			set_author(1, 1, 100);
```

Comment:
Not sure this makes sense since we are calling `roll_to_round_begin` with `2` below. It the test only works with this line here, it may be indicative of a bug somewhere.

Reply:
If there is no author for the first round, there is zero ponts for this round and then no inflation, with induce less rewards on future rounds because the inflation is a proportion of the existing supply